webapp/lib/model.yaml

```diff
@@ -7,6 +7,7 @@ Conversation:
   unread: 'publishable bool, true'
 
 Message:
+  firebaseDocId: 'none'
```

Comment:
What does this do?

Reply:
This tells codegen that `Message` is not a document but rather an element in a document and thus the `fromSnapshot` and `listen` methods need not be generated. I've updated the [model codegen doc](https://docs.google.com/document/d/1Jnr6TRjZFIlyw_pVXIwB58aGmUxkcWWYasOVt_jL42E/edit) with this information, but sometime soon I need to move this into a markup file in the `Infrastructure` repo

Reply:
Ooo a design doc, sweet :)
Great, thanks for this, that makes sense!